lib/internal/bootstrap/loaders.js

```diff
@@ -294,7 +294,7 @@
       const requireFn = this.id.startsWith('internal/deps/') ?
         NativeModule.requireForDeps :
         NativeModule.require;
-      fn(this.exports, requireFn, this, process);
+      fn(this.exports, requireFn, this, process, internalBinding);
```

Comment:
IIRC, it should be safe to remove `internalBinding` from `loaderExports` by now?

Reply:
bootstrap/node.js uses it, so we can't